pkg/handlers/archive.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var (
 	maxDepth   = 5
-	maxSize    = 250 * 1024 * 1024 // 250 MB
+	maxSize    = 2 << 30 // 2 GB
```

Comment:
Reverted this to the old value. This should no longer be an issue since we **do not** read the entire file into memory.

Reply:
I considered removing it, but to preserve backward compatibility, I left it in. This value can be configured via a command-line flag, so removing it isn't straightforward. 😞